daphne_server/src/roles/aggregator.rs

```diff
@@ -396,19 +396,31 @@ impl DapReportInitializer for crate::App {
         _part_batch_sel: &PartialBatchSelector,
         consumed_reports: Vec<EarlyReportStateConsumed>,
     ) -> Result<Vec<EarlyReportStateInitialized>, DapError> {
+        let valid_report_range = self.valid_report_time_range();
+
         tokio::task::spawn_blocking({
             let vdaf_config = task_config.vdaf;
             let vdaf_verify_key = task_config.vdaf_verify_key.clone();
             move || {
                 consumed_reports
                     .into_par_iter()
                     .map(|consumed_report| {
-                        EarlyReportStateInitialized::initialize(
-                            is_leader,
-                            &vdaf_verify_key,
-                            &vdaf_config,
-                            consumed_report,
-                        )
+                        let metadata = consumed_report.metadata();
+                        let initialized = if metadata.time < valid_report_range.start {
+                            consumed_report
+                                .into_initialized_rejected_due_to(TransitionFailure::ReportDropped)
+                        } else if metadata.time > valid_report_range.end {
+                            consumed_report
+                                .into_initialized_rejected_due_to(TransitionFailure::ReportTooEarly)
+                        } else {
```

Comment:
Do we also need to check if the timestamp is passed the tasks expiration date, or do we check this elsewhere?